app/server/ruby/lib/sonicpi/util.rb

```diff
@@ -130,9 +149,17 @@ def host_platform_desc
         if raspberry_pi_1?
           "Raspberry Pi 1"
         elsif raspberry_pi_2?
-          "Raspberry Pi 2"
+          "Raspberry Pi 2B"
         elsif raspberry_pi_3?
-          "Raspberry Pi 3"
+          "Raspberry Pi 3B"
+        elsif raspberry_pi_3bplus?
+          "Raspberry Pi 3B+"
+        elsif raspberry_pi_4_1gb?
+          "Raspberry Pi 4B:1gB"
```

Comment:
You have a little typo here:
```suggestion
          "Raspberry Pi 4B:1Gb"
```